ts/screens/modal/IdentificationModal.tsx

```diff
@@ -1,24 +1,32 @@
-import * as React from "react";
-import { useCallback, useState, useRef, useMemo, memo } from "react";
 import {
-  H2,
-  VSpacer,
-  Pictogram,
-  IconButton,
-  ContentWrapper,
   ButtonLink,
+  ContentWrapper,
+  H2,
+  IOPictograms,
   IOStyles,
+  IconButton,
+  Pictogram,
   ToastNotification,
-  IOPictograms
+  VSpacer
```

Comment:
Do you have some sort of lint for imports that shuffle them when you save the file? This leads to changes that are not actually needed and add entropy to PRs. Wouldn't be better not to see this unnecessary changes? What do you think?

Reply:
Got through `"source.organizeImports": true` in the VS Code IDE. It sorts the imports alphabetically and removes the unused ones.

Reply:
Got it, but if not all the team has it on, this adds only more entropy to the PR. So I suggest talk about the possibility to have this settings on for the all team, or not at all. What do you think?

Reply:
I agree, we should discuss this topic at the next team sync meeting 👍🏻